### DISCUSSION TITLE
不加速能打开steam商店 加速了反而打不开了

### DISCUSSION THREAD

**Discussion by AlanClemens** [BeyondDimension/SteamTools#3087] [Q&A]
rt

**Comment by pjy612** [comment]
黑人问号， 你应该是想说 加速了 却 **打不开** 吧？

**Comment by AlanClemens** [comment]
对对 打错了
________________________________
发件人: ***@***.*** ***@***.***> 代表 pjy612 ***@***.***>
发送时间: 2023年12月15日 22:58
收件人: BeyondDimension/SteamTools ***@***.***>
抄送: AlainPiso ***@***.***>; Author ***@***.***>
主题: Re: [BeyondDimension/SteamTools] 不加速能打开steam商店 加速了反而打开了 (Discussion #3087)


黑人问号， 你应该是想说 加速了 却 打不开 吧？

―
